web/modules/custom/nys_comment/nys_comment.routing.yml

```diff
@@ -5,7 +5,7 @@ comment.ban_user:
     _controller: '\Drupal\nys_comment\Controller\CommentController::banUser'
     entity_type: 'comment'
   requirements:
-    _permission: 'administer users'
+    _permission: 'administer users+administer comments'
```

Comment:
@aheaphy can you confirm that a comment moderator can ban a user, but not unban them?

Reply:
@routinet the user would have to have administer users permission, which the comment moderator role does not, if you tested as comment moderator you would see that you have no access to admin/people